src/kirby/components/list/list.component.tns.ts

```diff
@@ -0,0 +1,90 @@
+import { registerElement } from 'nativescript-angular';
```

Comment:
We shouldn't branch out on components unless it's absolutely necessary. Couldn't the differences between web and {N} be delegated to a helper service?